src/core/render-biblio.js

```diff
@@ -4,9 +4,33 @@
 import { addId } from "./utils";
 import hyperHTML from "hyperhtml";
 import { pub } from "./pubsubhub";
+import { lang as defaultLang } from "../core/l10n"
 
 export const name = "core/render-biblio";
 
+const localization_strings = {
```

Comment:
We tend not to use "_" in variable names... stick to camel case. 
 
```suggestion
const localizationStrings = {
```